install/README.md

```diff
@@ -24,6 +24,7 @@ Knative components on the following platforms:
 * [Knative Install on OpenShift](Knative-with-OpenShift.md)
 * [Knative Install on Minishift](Knative-with-Minishift.md)
 * [Knative Install on Pivotal Container Service](Knative-with-PKS.md)
+* [Knative Install on IBM Cloud Private](Knative-with-ICP.md)
```

Comment:
This list is in alphabetical order; please insert after IBM Cloud Kubernetes Service.